Examples/Catalog/Core/Source/Actions/NavigateAction.swift

```diff
@@ -0,0 +1,14 @@
+import Pilot
+
+public struct NavigateAction: Action {
```

Comment:
Style idea, feel free to take or leave, but if the entirety of this action is going to just be the destination it could just be an enum

```
public enum NavigateAction: Action {
    case topic(Topic)
    case modelCollectionExample(ModelCollectionExample)
}
```